Database/Corrections/Automatic/sodBaseQuests.lua

```diff
@@ -470,6 +470,15 @@ function SeasonOfDiscovery:LoadBaseQuests()
             [questKeys.requiredRaces] = raceIDs.NONE,
             [questKeys.requiredClasses] = classIDs.MAGE,
         },
+        [78144] = {
+            [questKeys.name] = "Alonso the Dragonslayer",
+            [questKeys.startedBy] = {{210995}},
+            [questKeys.finishedBy] = {{210995,}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 25,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.WARRIOR,
```

Comment:
```suggestion
            [questKeys.requiredClasses] = classIDs.WARRIOR,
            [questKeys.objectivesText] = {"Accompany Alonso to slay the dragon."},
            [questKeys.objectives] = nil,
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5416/commits/b251e6416f0fb61b5e42593546b490594eba3c7b